h/sentry.py

```diff
@@ -7,6 +7,22 @@
 import sentry_sdk.integrations.pyramid
 
 
+def _before_send(event, hint):
+    logger = event.get("logger")
+    message = event.get("logentry", {}).get("message")
```

Comment:
Some defensive programming here because the `event` dict doesn't always contain `"logger"` and `"logentry"`, it only contains them when the event is a logged error message, not when it's a raised exception. For exceptions `logger` and `message` will both be `None`.